astropy/units/tests/test_quantity_non_ufuncs.py

```diff
@@ -2078,4 +2088,4 @@ def test_all_included(self):
     # untested_function is created using all_wrapped_functions
     @needs_array_function
     def test_ignored_are_untested(self):
-        assert IGNORED_FUNCTIONS == untested_functions
+        assert IGNORED_FUNCTIONS == (untested_functions - TOSUPPORT_FUNCTIONS)
```

Comment:
I'd write this as `assert IGNORED_FUNCTIONS | TOSUPPORT_FUNCTIONS == untested_functions`

Reply:
✅